packages/ui-library/src/elements/toast/docs/component.md

```diff
@@ -0,0 +1 @@
+The Toast component is a general-purpose toast element which can be used to implement custom notifications.
```

Comment:
Here I would describe it as: 

The `Toast` component is a versatile toast element that can be used to implement custom notifications.

Reply:
Counter-proposal 😁 What about:
The `Toast` element is a versatile toast component that can be used to implement custom notifications.
In the original description I forgot to change `component` to `element`, as the `Toast` is now an `element`